warrior/Tools/reporting/html_results_template.html

```diff
@@ -622,7 +593,8 @@ <h2>Results Summary</h2>
 		<tr id="tr1" name="HeadingRow">
 			<th>Type</th>
 			<th>Name</th>
-			<th>Info</th>
+            <th>Info</th>
```

Comment:
Though Indentation is not a problem with xml. It helps to improve readability. Please consider right intending.